src/test/java/com/microsoft/sqlserver/jdbc/fmtOnly/APITest.java

```diff
@@ -0,0 +1,75 @@
+package com.microsoft.sqlserver.jdbc.fmtOnly;
+
+import static org.junit.Assert.assertEquals;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ParameterMetaData;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.UUID;
+
+import javax.sql.DataSource;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.microsoft.sqlserver.jdbc.SQLServerPreparedStatement;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractTest;
+
+
+public class APITest extends AbstractTest {
+
+    private static final String tableName = "FMT_API_Test_" + UUID.randomUUID();
+
+    @BeforeAll
+    private static void setupTest() throws SQLException {
+        try (Statement s = AbstractTest.connection.createStatement()) {

```

Comment:
`AbstractTest.connection` can just be `connection` since APITest is a child of AbstractTest.